Element/Digitizer.php

```diff
@@ -242,7 +242,9 @@ public function httpAction($action)
                 if (empty($request['dataItem'][ $uniqueIdKey ])) {
                     unset($request['dataItem'][ $uniqueIdKey ]);
                 }
-                $results = $dataStore->save($dataItem);
+
+                $dataIems = $dataStore->save($dataItem);
+                $results = ($dataIems->toArray());
 
```

Comment:
the returned result was a php object and was not serialized for the frontend, so no data was accessible in the frontend. Conversion to array solves this problem